CITATION.cff

```diff
@@ -0,0 +1,172 @@
+    affiliation:
+      National Institutes of Health, Scientific and Statistical Computing Core,
+      NIMH, Bethesda, MD 20892, USA
+    email: gangchen@mail.nih.gov
+
+  - family-names: Dafflon
+    given-names: Jessica
+    website: https://jessicadafflon.github.io/
+
+  - family-names: Nielson
+    given-names: Dylan
+    affiliation:
+      Machine Learning Team at the National Institute of Mental Health
+    website: https://github.com/Shotgunosine
+
+  - family-names: Esteban
+    given-names: Oscar
+    website: http://www.axonlab.org
+    orcid: https://orcid.org/0000-0001-8435-6191
+    affiliation:
+      Department of Radiology, Lausanne University Hospital, Lausanne, CH-1011,
+      Switzerland
+    email: phd@oscaresteban.es
+
+  - family-names: Gau
+    given-names: Rémi
+    affiliation: Université catholique de Louvain
+    email: remi.gau@uclouvain.be
+    orcid: "https://orcid.org/0000-0002-1535-9767"
+
+  - family-names: Kent
+    given-names: James
+    affiliation:
+      Psychoinformatics Laboratory at the University of Texas at Austin
+    website: https://jdkent.github.io/
+    email: jamesdkent21@gmail.com
+
+  - given-names: Camille
+    family-names: Maumet
+    website: https://camillemaumet.com/
+    orcid: https://orcid.org/0000-0002-6290-553X
+    affiliation: Inria, Univ Rennes, CNRS, Inserm, Rennes, 35042, France
+
+  - family-names: Mumford
+    given-names: Jeanette
+    website: https://github.com/jmumford
+    email: jeanette.mumford@gmail.com
+
+  - family-names: Narayan
+    given-names: Manjari
+    email: manjari@alumni.rice.edu
+
+  - family-names: Nichols
+    given-names: Thomas E. 
+    website: http://nisox.org
+    orcid: https://orcid.org/0000-0002-4516-5103
+    affiliation: University of Oxford, Big Data Institute, Oxford, OX3 7LF, UK
+    email: thomas.nichols@BDI.OX.AC.UK
+
+  - given-names: Hernando
+    family-names: Ombao
+    email: hernando.ombao@kaust.edu.sa
+
+  - family-names: Guiomar Niso Galán
+    given-names: Julia
+    website: http://guiomarniso.com
+    orcid: https://orcid.org/0000-0001-5872-8924
+    affiliation: Indiana University
+    email: guiomar.niso@ctb.upm.es
+
+  - family-names: Poldrack
+    given-names: Russell A. 
+    website: http://www.poldracklab.org
+    orcid: https://orcid.org/0000-0001-6755-0259
+    affiliation: Stanford University, Stanford, CA 94305 USA
+    email: russpold@stanford.edu
+
+  - family-names: Poline
+    given-names: Jean-Baptiste
+    website: https://github.com/jbpoline
+    orcid: https://orcid.org/0000-0002-9794-749X
+
+  - family-names: Thirion
+    given-names: Bertrand
+    website: https://github.com/bthirion
+    orcid: https://orcid.org/0000-0001-5018-7895
+
+  - family-names: de la Vega
+    given-names: Alejandro
+    website: https://github.com/adelavega
+
+  - family-names: Yarkoni
+    given-names: Tal
+    website: https://github.com/tyarkoni
+    orcid: https://orcid.org/0000-0002-6558-5113
```

Comment:
```suggestion
    orcid: https://orcid.org/0000-0002-6558-5113
   
  - family-names: Halchenko
    given-names: Yaroslav
    affiliation: Dartmouth College
    website: https://centerforopenneuroscience.org/whoweare#yaroslav_o_halchenko_
    orcid: "https://orcid.org/0000-0003-3456-2493"
```

I believe I've participated in some meetings etc in the past.

Reply:
I'll add in a commit.